### PR TITLE
Updated da.po

### PR DESCRIPTION
**Pull request by matsduf**
* All fuzzy translated replaced by English
* Preparation for translation to Danish

When this is merged, an issue for translation to Danish will be created.

### CODE REVIEW

File: share/da.po [hunk 1]
```diff
@@ -130,15 +128,15 @@ msgstr "Alle navneservernes IP-adresser er unikke."
 
 #. CHILD_DISTINCT_NS_IP
 #: ../lib/Zonemaster/Engine/Test/Delegation.pm:127
-#, fuzzy
+#,
```
Comment (mattias-p) [inline share/da.po]: May I ask what tool(s) you used to make these updates? Just curious.

File: share/da.po [hunk 2]
```diff
@@ -600,12 +578,12 @@ msgid "IP {address} refers to multiple nameservers ({nss})."
 msgstr "IP-adressen {address} refererer til flere navneservere ({nss})."
 
 #. IPV4_DISABLED
+#: ../lib/Zonemaster/Engine/Test/Nameserver.pm:288
```
Comment (mattias-p) [inline share/da.po]: I'm bothered by how these line numbers jump around. They're clearly not meant to be checked in. I'll create an issue to replace them with some other module/test case annotations.

File: share/da.po (hunk 1 above)
Comment (matsduf) [reply share/da.po]: Just emacs.

File: share/da.po (hunk 2 above)
Comment (mattias-p) [reply share/da.po]: Created #625.

File: share/da.po (hunk 1 above)
Comment (mattias-p) [reply share/da.po]: PO mode or vanilla?

Comment (matsduf) [reply share/da.po]: PO mode. I felt that the risk errors was lower.

**Pull request merged**